File: 01936.py
SCISSORS = 1; ROCK = 2; PAPER = 3

A, B = map(int, input().split())

if A == SCISSORS:
    if B == ROCK: print("B")
    else: print("A")
elif A == ROCK:
    if B == SCISSORS: print("A")
    else: print("B")
else:
    if B == SCISSORS: print("B")
    else: print("A")

File: 01936.cpp
#include <iostream>
using namespace std;

#define SCISSORS 1
#define ROCK 2
#define PAPER 3

int main() {
	int A, B; cin >> A >> B;

	if (A == SCISSORS) {
		if (B == ROCK) cout << "B\n";
		else cout << "A\n";
	}
	else if (A == ROCK) {
		if (B == SCISSORS) cout << "A\n";
		else cout << "B\n";
	}
	else {
		if (B == SCISSORS) cout << "B\n";
		else cout << "A\n";
	}
}
